src/controller/CHIPCommissionableNode.h

```diff
@@ -0,0 +1,101 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2013-2017 Nest Labs, Inc.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      Declaration of CHIP CommissionableNode
+ *
+ */
+
+#pragma once
+
+#include <mdns/Resolver.h>
+#include <support/logging/CHIPLogging.h>
+
+namespace chip {
+
+namespace Controller {
+
+constexpr uint16_t kMdnsPort = 5388;
+
+// struct ResolvedNodeData;
+struct CommissionableNodeData;
```

Comment:
why do we need forward declaration? Can we #include the relevant header instead?

Reply:
Done. Removed.